client/gutenberg/editor/api-middleware/index.js

```diff
@@ -89,10 +89,17 @@ export const wpcomPathMappingMiddleware = ( options, next, siteSlug ) => {
 };
 
 const wpcomRequest = method => {
-	if ( method !== 'GET' ) {
-		return wpcom.req.post.bind( wpcom.req );
+	/* wpcom.req.del is just a wrapper around wpcom.req.post
+	 * and it always uses the `POST` method.
+	 * https://github.com/Automattic/wpcom.js/blob/master/lib/util/request.js#L70
+	 *
+	 * Instead we use wpcom.req.get for `DELETE`, that passes the method
+	 * along with the rest of the request parameters.
+	 */
+	if ( includes( [ 'GET', 'DELETE' ], method ) ) {
```

Comment:
two things: we can use `Array.protype.includes` here thanks to our Babel transform

why are we binding `DELETE` to a `GET`?

> that passes the method along with the rest of the request parameters

are you talking about an HTTP envelope? I'm not seeing that anywhere here in the code.